index.js

```diff
@@ -435,5 +435,17 @@ export const mentorshipTeam = [
 			},
 		],
 	},
-
+	{
+                name: 'Quynh Tran',
+                title: 'Student @ DePauw University',
+                hobbies: 'sleep, study, travel',
+                avatar: '',
```

Comment:
If you are not comfortable with displaying your pic on the website, use this one `https://res.cloudinary.com/khoa165/image/upload/v1711083640/viettech/placeholder.jpg`